buildSrc/src/main/java/org/opensearch/gradle/test/DistroTestPlugin.java

```diff
@@ -77,7 +77,7 @@
 public class DistroTestPlugin implements Plugin<Project> {
     private static final String SYSTEM_JDK_VERSION = "8u242+b08";
     private static final String SYSTEM_JDK_VENDOR = "adoptopenjdk";
-    private static final String GRADLE_JDK_VERSION = "17.0.2+8";
+    private static final String GRADLE_JDK_VERSION = "11.0.15+10";
```

Comment:
I presume this is unused since it didn't break anything?

Reply:
To be honest, it seems like the version is used in some tests but not JDK itself, thanks!